pages/profile.tsx

```diff
@@ -46,6 +51,79 @@ const Profile: NextPage<PageProps> = (props) => {
     [setSelectedMarket]
   );
 
+  const handleSubmitModifyForm = React.useCallback(
+    async ({
+      email_sub,
+      email,
+      username,
+    }: {
+      username?: string;
+      email?: string;
+      email_sub: boolean;
```

Comment:
I guess this is acceptable, because it would make more mess to change it to camelCase